frontend/packages/dev-console/src/components/pipelines/pipelinesResources/PipelinesResourceForm.tsx

```diff
@@ -46,15 +46,19 @@ const PipelineResourceForm: React.FC<PipelineResourceFormProps> = ({
       type: 'cluster',
       param: {
```

Comment:
we could use `params` instead of `param`
```suggestion
      params: {
```